appengine/standard_python37/bigquery/main.py

```diff
@@ -35,20 +36,40 @@ def main():
         WHERE tags like '%google-bigquery%'
         ORDER BY view_count DESC
         LIMIT 10
-    """)
+    """
+    )
+
+    return flask.redirect(
+        flask.url_for(
+            "results",
+            project_id=query_job.project,
+            job_id=query_job.job_id,
+            location=query_job.location,
+        )
+    )
+
+
+@app.route("/results")
+def results():
+    project_id = flask.request.args.get("project_id")
+    job_id = flask.request.args.get("job_id")
+    location = flask.request.args.get("location")
+
+    bigquery_client = bigquery.Client(project=project_id, location=location)
+    query_job = bigquery_client.get_job(job_id)
```

Comment:
`get_job` takes a `project` and `location` argument. (See [reference](https://googleapis.github.io/google-cloud-python/latest/bigquery/generated/google.cloud.bigquery.client.Client.html#google.cloud.bigquery.client.Client.get_job)). You don't need to create a client in the line above. Instead, pass the `project` and `location` here to `get_job` on the existing (global) `bigquery_client`.

Reply:
Removed line 58 and changed to:
`query_job = bigquery_client.get_job(job_id, project=project_id, location=location)` as suggested!